web/src/locales/en.json

```diff
@@ -139,5 +139,10 @@
 	"employees.updatePassword.error.incorrectPassword.description": "The password entered does not match your current password.",
 	"employees.error.passwordMismatch": "The password doesn't match the new password",
 	"employees.passwordConstraints": "Password must contain at least 14 characters, including one regular character, one special character, and one number",
-	"confirm": "Confirm"
+	"confirm": "Confirm",
+	"employees.update.success": "Employee updated successfully",
+	"employees.username.placeholder": "Enter username",
+	"employees.firstName.placeholder": "Enter first name",
+	"employees.lastName.placeholder": "Enter last name",
+	"employees.phone.placeholder": "Enter phone number"
```

Comment:
This seems to be in the wrong place, after the merge.
Shouldn't it be place after `employees.delete.conflict.description`?